index.d.ts

```diff
@@ -275,6 +259,37 @@ type PageTypeType = {
 	isSensitive: boolean;
 };
 
+// Data types for the API request bodies from clients. The 'CAPI' prefix
+// convention doesn't really make sense anymore (it is not the CAPI model,
+// though closely related) but is not worth changing now.
```

Comment:
This might be worth revisiting. The CAPI prefix is meant to denote data that came from CAPI (or rather, Frontend) and that has a _different_ type to the one eventually passed down the react stack. We now use `decideThing` functions to transform any data we get from api calls. This is mostly related to `CAPIPillar`; the idea was to try to make any later refactor easier if and when Frontend adopts `Format`.

I _think_ that this is also the case here? I just wanted to double check

Reply:
@oliverlloyd okay thanks. I'll tweak the comment to include the 'and has a different type to our internal use' caveat.